organization/summit.md

```diff
@@ -0,0 +1,23 @@
+# Mentor Summit
+
+Every year Google promote a Summit
+and each [**mentoring**] organization,
+**in this case only NumFOCUS**,
+may send two (2) members to the Mentor Summit.
+
+NumFOCUS will nominate as Summit ambassador
+the lead administrator of the current year
+**and** the lead administrator of following year.
```

Comment:
I don't really know a better solution. I would give it a try and see if it works out. My idea to find a new admin would be to invite a member of the participating orgs if he wants to be the secondary admin and help me to look over the applications / student gradings. That way we have already experience with him and active project members should be likely to stay on GSoC for more then a year.